go.mod

```diff
@@ -7,34 +7,34 @@ require (
 	github.com/golang-jwt/jwt v3.2.2+incompatible
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/google/uuid v1.3.0
-	github.com/gookit/color v1.5.1
+	github.com/gookit/color v1.5.2
 	github.com/mssola/user_agent v0.5.3
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v1.5.0
-	github.com/spf13/viper v1.12.0
+	github.com/spf13/cobra v1.6.0
```

Comment:
@igorlombacx  Lets remove these updates, we have dependabot on the project and the bumps should be handled by that.